pusher-platform-android/src/main/AndroidManifest.xml

```diff
@@ -1,6 +1,6 @@
 <manifest xmlns:android="http://schemas.android.com/apk/res/android"
     package="com.pusher.platform.sdk" >
 
-    <uses-permission android:name="android.permission.ACCESS_NETWORK_STATE" />
+    <uses-permission android:name="android.permission.INTERNET" />
```

Comment:
The old permission was needed for the Connectivity Check. Now not needed, so replaced it with one that would always need, internet